### ISSUE TITLE
Consolidate stash and gitlab to establish single solution

### ISSUE THREAD

**Issue by pzim** [pzim/reaktor#13]
The gitlab and stash code are very similar and should be consolidated into a single solution to remove the redundant code.

**Comment by Tokynet** [comment]
Is there any known implementation with stash?

**Comment by pzim** [comment]
Yes - reaktor will work with stash payloads. You can see an example of what the webhook url would look like for stash (stash_payload) in the README.

**Comment by Tokynet** [comment]
Thanks for the quick reply,
Cheers!
